opennms-webapp-rest/src/test/resources/v1/alarms.json

```diff
@@ -21,7 +21,7 @@
                 "nodeId": 1, 
                 "nodeLabel": "node1", 
                 "parameters": [
-                    {"name": "testParm", "value": "HelloWorld", "type":"string" }
+                    {"name": "testParm", "value": "HelloWorld", "type":"string", "position": 0}
```

Comment:
@j-white the api exposes the database model (OnmsEvent) via rest and thus we have the new field "position" in the response. Are we ok with that or do we need to do something about it?

Reply:
I would hide it. Marking the attribute with `@XmlTransient` instead of `@XmlAttribute` should do the trick.

Reply:
:+1: makes sense